src/page/generate/ViewDocumentPage.tsx

```
@@ -230,14 +229,15 @@ Silakan lanjutkan pembuatan bagian berikutnya dari dokumen tanpa mengulangi kont
   }, [documentObject, isLoading, isGenerating, processingResponse, processDocumentObject]);
 
   // Save the generated document back to Supabase
-  const saveGeneratedDocument = async () => {
-    if (!documentId || generatedPages.length === 0) return;
+  const saveGeneratedDocument = async (pagesToSave?: any[]) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)